### PR TITLE
Feature std dev

### PR DESCRIPTION
**Pull request by FlorianGeerken**
Extend the graphs to show the daily std as required by S.R1.1.1

### CODE REVIEW

File: inflammation/models.py [hunk 1]
```diff
@@ -32,3 +32,13 @@ def daily_min(data):
...
+
+def s_dev(data):
+    """Computes and returns standard deviation for data."""
+    mmm = np.mean(data, axis=0)
+    devs = []
+    for entry in data:
+        devs.append((entry - mmm) * (entry - mmm))
+
+    s_dev2 = sum(devs) / len(data)
+    return {'standard deviation': s_dev2}
```
Comment (cptboris) [inline inflammation/models.py:44]: Why is a dictionary returned in this function?

File: inflammation-analysis.py [hunk 2]
```diff
@@ -21,7 +21,7 @@ def main(args):
     for filename in InFiles:
         inflammation_data = models.load_csv(filename)
 
-        view_data = {'average': models.daily_mean(inflammation_data), 'max': models.daily_max(inflammation_data), 'min': models.daily_min(inflammation_data)}
+        view_data = {'average': models.daily_mean(inflammation_data), 'max': models.daily_max(inflammation_data), 'min': models.daily_min(inflammation_data), **(models.s_dev(inflammation_data))}
```
Comment (cptboris) [inline inflammation-analysis.py:41]: Maybe I do not understand enough about python, but is leaving a **(models....) a good idea? Why not use a 'std_dev' key?

File: inflammation/models.py (hunk 1 above)
Comment (cptboris) [inline inflammation/models.py:73]: Why is this function returning a dictionary? The other functions here are returning numpy things, not dictionaries

File: tests/test_models.py [hunk 3]
```diff
@@ -29,3 +31,13 @@ def test_daily_mean_integers():
     # Need to use Numpy testing functions to compare arrays
     npt.assert_array_equal(daily_mean(test_input), test_result)
 
+
+@pytest.mark.parametrize('data, expected_standard_deviation', [
+    ([0, 0, 0], 0.0),
+    ([1.0, 1.0, 1.0], 0),
+    ([0.0, 2.0], 1.0)
+])
```
Comment (cptboris) [inline tests/test_models.py:116]: Maybe it is a good idea to include a test for negative input numbers? It should not happen as inflammation is not negative, but nevertheless it might be a good idea.

Let me know if you agree

File: tests/test_models.py [hunk 4]
```diff
@@ -1,7 +1,9 @@
 """Tests for statistics functions within the Model layer."""
 
+import os
```
Comment (cptboris) [inline tests/test_models.py]: We are not using os package in this testing_models for this file, does this import make sense here?

**Review by cptboris — COMMENTED** [review]
Thanks for the PR @FlorianGeerken! It is nice to see such contributions.

Could you please see my comments for this PR? I would appreciate your opinion on these.

File: inflammation-analysis.py (hunk 2 above)
Comment (FlorianGeerken) [reply inflammation-analysis.py:41]: Thanks for the input, I might not understand this function well enough myself :) Could you elaborate a bit more or maybe even add a suggestion? Much appreciated.

File: tests/test_models.py (hunk 3 above)
Comment (FlorianGeerken) [reply tests/test_models.py:116]: Fixed via 40702d1

File: inflammation-analysis.py (hunk 2 above)
Comment (cptboris) [reply inflammation-analysis.py:41]: I think that the last entry in this dictionary does not have a key. It might result in some issues down the line.

I think that a good idea might be to replace ** by 'std_dev': or something like that.

**Review by cptboris — APPROVED** [review]
As an exercise this is good enough. Let's have this completed.
